src/VisualStudio/CSharp/Test/F1Help/F1HelpTests.cs

```diff
@@ -174,7 +174,17 @@ public async Task TestPartialType()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.F1Help)]
-        public async Task TestPartialMethod()
+        public async Task TestRecordPartialType()
+        {
+            await Test_KeywordAsync(
+@"part[||]ial record C
```

Comment:
Would be good to have a test for a record with a primary constructor too

Reply:
Added test.